Lib/test/test_tk.py

```diff
@@ -5,7 +5,7 @@
 # Skip test if tk cannot be initialized.
 support.requires('gui')
 
-from tkinter.test import runtktests
+from Lib.test.tkinter_test import runtktests
```

Comment:
This can’t be correct.  `Lib` is not an importable Python package, it is a path containing modules and packages.
The new import should be `from test.tkinter_test import runtktests`.

(Or for what it’s worth, this nearly empty file could be removed, if the new package for tkinter tests would be named in a way that’s automatically found by unittest discover and runtests)

Reply:
Done, I think it might be better to separate the changes no? 
Like the first stage - migrate the module. Second stage - remove redundant legacy files.